docs/plugins.md

```diff
@@ -225,6 +225,7 @@ defaultOptions: {
 A `util` module from Prettier core is considered a private API and is not meant to be consumed by plugins. Instead, the `util-shared` module provides the following limited set of utility functions for plugins:
 
 ```ts
+createLanguage(linguistData: object, { extend: object, override: object }): object;
```

Comment:
If we're exporting it I think we should document better what `linguistData` is and where to get it from, as well as what `override` and `extend` mean.

Reply:
@azz i am bad in documentation on English, can you help me with this?